src/front-door/azext_front_door/custom.py

```diff
@@ -646,19 +646,22 @@ def update_fd_routing_rule(parent, instance, item_name, frontend_endpoints=None,
 def create_waf_policy(cmd, resource_group_name, policy_name,
                       disabled=False, mode=None, redirect_url=None,
                       custom_block_response_status_code=None,
-                      custom_block_response_body=None, tags=None):
+                      custom_block_response_body=None, tags=None,
+                      request_body_check=None, sku=None):
     client = cf_waf_policies(cmd.cli_ctx, None)
     from azext_front_door.vendored_sdks.models import (
-        WebApplicationFirewallPolicy, ManagedRuleSetList, PolicySettings, CustomRuleList)
+        WebApplicationFirewallPolicy, ManagedRuleSetList, PolicySettings, CustomRuleList, Sku, SkuName, PolicyRequestBodyCheck)
     policy = WebApplicationFirewallPolicy(
         location='global',
         tags=tags,
+        sku=Sku(name=sku if sku is not None else SkuName.classic_azure_front_door),
         policy_settings=PolicySettings(
             enabled_state='Enabled' if not disabled else 'Disabled',
             mode=mode,
             redirect_url=redirect_url,
             custom_block_response_status_code=custom_block_response_status_code,
-            custom_block_response_body=custom_block_response_body
+            custom_block_response_body=custom_block_response_body,
+            request_body_check=request_body_check if request_body_check is not None else PolicyRequestBodyCheck.disabled,
```

Comment:
get_three_state_flag will return disabled if use don't provide that parameter. You can see code for detail

Reply:
Sure, simplified it